model/metrics/gen-ai.yaml

```diff
@@ -0,0 +1,39 @@
+groups:
+  - id: metric_attributes.gen_ai
+    type: attribute_group
+    brief: 'This group describes GenAI metrics attributes'
+    attributes:
+      - ref: error.type
```

Comment:
could you please also reference `error.type`, `server.address` and `server.port` to spans as well?

Reply:
(also fine to do it in a separate PR since we are not adding operation to spans in this one)